src/js/component/typeahead.js

```diff
@@ -127,6 +127,7 @@ export class TypeaheadOmniboxController {
     }
 
     selectField(field) {
+        console.log(field);
```

Comment:
oops

Reply:
well the commit name and the PR's title start with a `WIP:` so maybe, just maybe, this PR is not ready to be merge

Reply:
not ready to be merge != not ready for feedback.
If you do not want early feedback, please mention it in order to avoid this kind of misunderstanding